batch2/Dockerfile.worker

```diff
@@ -2,7 +2,8 @@ FROM python:3.6-slim-stretch
 
 RUN apt-get update && \
   apt-get -y install \
-    curl && \
+    curl \
+    less && \
```

Comment:
You're using this in the worker container?

Reply:
Yes. It was irritating to not be able to see the worker.log with `less` while it was running so I added it for now, at least while we're still debugging things.